packages/docs/docs/guide/deploy.md

```diff
@@ -56,6 +56,35 @@ cd -
 You can also run the above script in your CI setup to enable automatic deployment on each push.
 :::
 
+### GitHub Pages and GitHub Actions
+
+1. Create a personal access token
+2. Create encrypted secrets under your repository
+3. Create a `.yml` or `.yaml` file in the `.github/workflows` directory in the root of your repository. E.g:`vueoress-deploy.yml`;
```

Comment:
```suggestion
3. Create a `.yml` or `.yaml` file in the `.github/workflows` directory in the root of your repository. E.g:`vuepress-deploy.yml`;
```

Reply:
Thank you for your careful review!! I recommited a version  according to the suggestion.
About the minimalist repos for testing, please see the comment below